src/main/java/net/sf/jabref/preferences/JabRefPreferences.java

```diff
@@ -303,7 +303,6 @@
     public static final String SHOW_FILE_LINKS_UPGRADE_WARNING = "showFileLinksUpgradeWarning";
     public static final String SIDE_PANE_WIDTH = "sidePaneWidth";
     public static final String LAST_USED_EXPORT = "lastUsedExport";
-    public static final String LAST_USED_IMPORT = "lastUsedImport";
```

Comment:
Why did you remove the Preference? It is a really nice feature to preselect the last used Filefilter for exporting/importing.